src/OrchardCore.Modules/OrchardCore.Contents/Handlers/FullTextAspectSettingsHandler.cs

```diff
@@ -72,12 +72,13 @@ public override Task GetContentItemAspectAsync(ContentItemAspectContext context)
                 if (settings.IncludeFullTextTemplate && !String.IsNullOrEmpty(settings.FullTextTemplate))
                 {
                     var templateContext = new TemplateContext();
+                    templateContext.SetValue("ContentItem", context.ContentItem);
```

Comment:
Shouldn't we decide?

Reply:
I mean next line defines `Model`

Reply:
You're right, it is just to allow to use `ContentItem` directly as in many other places.

In these other places it was the opposite we could only use `ContentItem` but not `Model.ContentItem`. By adding a model in these other places we can also use `Model.ContentItem`. So

- Do we need to remove the `SetValue("ContentItem")` everywhere because we now have the model? But it would break liquid expressions not using `Model.ContentItem`.

- Or let it as it is so that everywhere we can both use `ContentItem` and `Model.ContentItem`.

Note: There are some remaining places (as in workflows) where there is nothing to build a model so just a `ContentItem` is set. But i can build a model that only hold a `ContentItem` property, anyway i will do it and then also remove `SetValue("ContentItem")` depending on what we decide above.

Reply:
Oh yes i remember, just below we define the `ContentItem` as the model, but because `ContentItem` has a `ContentItem` property pointing to itself, we can use `Model.ContentItem` as in other places, another singularity ;)

That's why i also added `ContentItem` to allow e.g `ContentItem.DisplayText` as we can do in other places.

I will remove it here.